heroku/cli
# refactor(container-registry-v5): Move 'certs:info' to CLI

### Description

This converts `certs:info` to oclif and updates its associated test.

### Testing

1) Run `./bin/run certs:info -a APP` with this branch pulled down with an app that has a cert
2) Verify the command succeeds
3) Compare the results with prod cli command and make sure things look 👍
4) Repeat 1-3 with an app that has more than one cert and verify it works by specifying with `--name` flag
